common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -405,9 +405,13 @@ private void launchCompanyPortal() {
     private void processAmazonAppUri(@NonNull final String url) {
         final String methodTag = TAG + ":processAmazonAppUri";
 
-        final Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
-        getActivity().startActivity(intent);
-        Logger.info(methodTag, "Sent Intent to launch Amazon app");
+        try {
+            final Intent intent = new Intent(Intent.ACTION_VIEW, Uri.parse(url));
+            getActivity().startActivity(intent);
+            Logger.info(methodTag, "Sent Intent to launch Amazon app");
+        } catch (android.content.ActivityNotFoundException e) {
```

Comment:
Could consider catching all kinds of throwables here

```suggestion
        } catch (Throwable throwable) {
```